Applications/Spire/Include/Spire/Ui/TabWidget.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef SPIRE_TAB_WIDGET_HPP
+#define SPIRE_TAB_WIDGET_HPP
+#include <QTabWidget>
+
+namespace Spire {
+
+  //! Represents a Spire-styled QTabWidget.
+  class TabWidget : public QTabWidget {
+    public:
+
+      //! Constructs a TabWidget.
+      /*!
+        \param parent The parent widget.
+      */
+      TabWidget(QWidget* parent = nullptr);
```

Comment:
Mark ```explicit```, I know it's a very annoying C++ feature, just have to burn it into muscle memory.